pkg/integrations/oracledb_exporter/oracledb_exporter.go

```diff
@@ -0,0 +1,103 @@
+package oracledbexporter
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/grafana/agent/pkg/integrations"
+	oe "github.com/observiq/oracledb_exporter/collector"
+
+	// required driver for integration
+	_ "github.com/sijms/go-ora/v2"
+	"github.com/xo/dburl"
+)
+
+// DefaultConfig is the default config for the oracledb v2 integration
+var DefaultConfig = Config{
+	ConnectionString:     os.Getenv("DATA_SOURCE_NAME"),
+	MaxOpenConns:         10,
+	MaxIdleConns:         0,
+	QueryTimeout:         5,
+	MetricScrapeInterval: 0,
+}
+
+var (
+	errNoConnectionString = errors.New("no connection string was provided")
+)
+
+// Config is the configuration for the oracledb v2 integration
+type Config struct {
+	ConnectionString     string        `yaml:"connection_string"`
+	MaxIdleConns         int           `yaml:"max_idle_connections"`
+	MaxOpenConns         int           `yaml:"max_open_connections"`
+	MetricScrapeInterval time.Duration `yaml:"metric_scrape_interval"`
+	QueryTimeout         int           `yaml:"query_timeout"`
+}
+
+// Validate returnsif the configuration is valid
+func (c *Config) Validate() error {
+	if c.ConnectionString == "" {
+		return errNoConnectionString
+	}
+	if _, err := dburl.Parse(c.ConnectionString); err != nil {
```

Comment:
Looks to me, based on the included tests, like the connection string needs to be in a fairly specific format. May be prudent to add some extra checks here to ensure it's not invalid?

E.g. I could see a case here where a valid URL scheme is supplied, but one which is missing the oracle-specific formatting. 

If this isn't actually required, then i'd like to see a test or comment added to address that case 

Same applies for v2